README.md

```diff
@@ -188,7 +188,7 @@ While full, compiled examples are available in
 a common usage example is shown here. This example assumes that you are writing
 a client for a JSON API representing a person that looks like the following:
 
-```
+```js
```

Comment:
I went with JS since this isn't valid JSON.

JSON:
```js
{
  id: 12345,
  age: 35,
  first_name: 'Jimmy',
  hobbies: ['jumping', 'basketball']
}
```

js:
```json
{
  id: 12345,
  age: 35,
  first_name: 'Jimmy',
  hobbies: ['jumping', 'basketball']
}
```

I can revert this back to unhighlighted if that is preferred.

Reply:
Thanks for the PR!

I guess to be valid JSON the keys should be strings? I think that was actually what was intended. (Certainly, the keys _do_ need to be strings). Does that make it happy with 'json'?

Reply:
Sure thing!

Yup, I just pushed up a commit that switches it to JSON syntax/highlighting